packages/trader/src/Modules/Contract/Components/AccumulatorsStats/accumulators-stats-manual-modal.tsx

```diff
@@ -55,30 +43,22 @@ const AccumulatorsStatsManualModal = ({
                 <Modal.Body className='accumulators-stats-modal-body'>
                     <div className='accumulators-stats-modal-body__video'>
                         {is_loading && <Loading is_fullscreen={false} />}
-                        <video
-                            autoPlay
-                            data-testid='dt_accumulators_stats_manual_video'
+                        <VideoStream
+                            autoplay
+                            test_id='dt_accumulators_stats_manual_video'
                             loop
-                            onLoadedData={() => setIsLoading(false)}
-                            playsInline
```

Comment:
when video is loaded from CF Stream inside iframe, `playsInline` is enabled by default, hence removing it